mirascope/wandb/prompt.py

```diff
@@ -29,22 +29,31 @@ class WandbPrompt(BasePrompt):
         start_time_ms=round(datetime.datetime.now().timestamp() * 1000),
         metadata={"user": "mirascope_user"},
     )
-    chat = OpenAIChat(api_key="YOUR_OPENAI_API_KEY")
 
     class HiPrompt(WandbPrompt):
     """{greeting}."""
 
     greeting: str
 
-    prompt = HiPrompt(span_type="llm", greeting="Hello")
-    completion = chat.create(prompt)
+    prompt = HiPrompt(
+        api_key="YOUR_OPENAI_API_KEY",
+        span_type="llm",
+        greeting="Hello",
+    )
+    completion = prompt.create()
```

Comment:
What if we updated the `WandbPrompt.create` method so it looked like this:
```python
completion, span = prompt.create(parent_span)
```

Note that both `trace` and `trace_error` would then happen automatically, right?